### PR TITLE
Fix the path doc for AddFile and RemoveFile

### PR DESCRIPTION
**Pull request by zsxwing**
Mention that the path is encoded.

### CODE REVIEW

File: PROTOCOL.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ The schema of the `remove` action is as follows:
 
 Field Name | Data Type | Description
 -|-|-
-path | String | An absolute or relative path to a file that should be removed from the table
+path| String | A relative path to a file from the root of the table, or an absolute path to a file, that should be added to the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt) which needs to be decoded properly to get the file path.
```
Comment (brkyvz) [inline PROTOCOL.md:244]: should be removed from the table

Comment (brkyvz) [reply PROTOCOL.md:244]: also "decoded" instead of "decoded properly"

File: PROTOCOL.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ The schema of the `add` action is as follows:
 
 Field Name | Data Type | Description
 -|-|-
-path| String | A relative path, from the root of the table, to a file that should be added to the table
+path| String | A relative path to a file from the root of the table, or an absolute path to a file, that should be added to the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt) which needs to be decoded properly to get the file path.
```
Comment (brkyvz) [inline PROTOCOL.md:218]: ditto, "properly" is not redundant

File: PROTOCOL.md [hunk 3]
```diff
@@ -225,7 +225,7 @@ The schema of the `add` action is as follows:
 
 Field Name | Data Type | Description
 -|-|-
-path| String | A relative path, from the root of the table, to a file that should be added to the table
+path| String | A relative path to a file from the root of the table, or an absolute path to a file, that should be added to the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt) which needs to be decoded to get the file path.
```
Comment (brkyvz) [inline PROTOCOL.md:228]:
```suggestion
path| String | A relative path to a file from the root of the table, or an absolute path to a file that should be added to the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt), which needs to be decoded to get the file path.
```

File: PROTOCOL.md [hunk 4]
```diff
@@ -251,7 +251,7 @@ The schema of the `remove` action is as follows:
 
 Field Name | Data Type | Description
 -|-|-
-path | String | An absolute or relative path to a file that should be removed from the table
+path| String | A relative path to a file from the root of the table, or an absolute path to a file, that should be removed from the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt) which needs to be decoded to get the file path.
```
Comment (brkyvz) [inline PROTOCOL.md:254]:
```suggestion
path| String | A relative path to a file from the root of the table, or an absolute path to a file that should be removed from the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt), which needs to be decoded to get the file path.
```

**Review by brkyvz — APPROVED** [review]
LGTM

File: PROTOCOL.md [hunk 5]
```diff
@@ -225,7 +225,7 @@ The schema of the `add` action is as follows:
 
 Field Name | Data Type | Description
 -|-|-
-path| String | A relative path, from the root of the table, to a file that should be added to the table
+path| String | A relative path to a file from the root of the table, or an absolute path to a file that should be added to the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt), which needs to be decoded to get the file path.
```
Comment (jaceklaskowski) [inline PROTOCOL.md:228]: nit: Is the other comma (after `the table`) needed?

File: PROTOCOL.md [hunk 6]
```diff
@@ -251,7 +251,7 @@ The schema of the `remove` action is as follows:
 
 Field Name | Data Type | Description
 -|-|-
-path | String | An absolute or relative path to a file that should be removed from the table
+path| String | A relative path to a file from the root of the table, or an absolute path to a file that should be removed from the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt), which needs to be decoded to get the file path.
```
Comment (jaceklaskowski) [inline PROTOCOL.md:254]: same here

**Review by jaceklaskowski — APPROVED** [review]

**Comment by homar** [comment]
Why this wasn't merge? Is there any other pr that actually fix this part of the doc?

**Comment by zsxwing** [comment]
Reopening this. This has not yet been merged.

**Pull request closed without merging**